clidocstool_yaml.go

```diff
@@ -112,6 +113,20 @@ func (c *Client) genYamlTreeCustom(cmd *cobra.Command, filePrepender func(string
 		return nil
 	}
 
+	// inherit parents code delimiter annotations if not defined for the
+	// current cmd
+	cmd.VisitParents(func(c *cobra.Command) {
+		if _, ok := cmd.Annotations[annotation.CodeDelimiter]; ok {
+			return
+		}
+		if cd, ok := c.Annotations[annotation.CodeDelimiter]; ok {
+			if cmd.Annotations == nil {
+				cmd.Annotations = map[string]string{}
+			}
+			cmd.Annotations[annotation.CodeDelimiter] = cd
+		}
+	})
+
```

Comment:
Could you combine this with the existing loop in https://github.com/docker/cli-docs-tool/blob/383716cbaeec9ec51f16be5765aed9115f8236a8/clidocstool_yaml.go#L168-L188

(also starting to wonder if that should be combined / moved to a shared (yaml + markdown) location, but not all of these may be needed for MarkDown 🤔

Reply:
> (also starting to wonder if that should be combined / moved to a shared (yaml + markdown) location, but not all of these may be needed for MarkDown 🤔

Yes we might do that in a follow-up.